### DISCUSSION TITLE
Boat for lava

### DISCUSSION THREAD

**Discussion by cardanawandra** [PurpurMC/Purpur#494] [Feature Requests]
allow boat for lava driving like water does

**Comment by kohy-creates** [comment]
Maybe a config where it allows to do so if the boat has fire resistance
Or a specific NBT tag, like Purpur.allowLavaDriving

**Comment by RhythmicSys** [comment]
I have tried to implement this idea, but ran into issues where the boat constantly sinks. I am not sure if this is solvable within the server code, or if it would require a plugin instead
